docs/source/dev_guide/algorithms/optimization-solvers/solvers/stochastic-gradient-descent-algorithm.rst

```diff
@@ -124,79 +124,83 @@ method parameter method:
      - Default Value
      - Description
    * - algorithmFPType
-     - defaultDense, miniBatch, or momentum
-     - float
+     - ``defaultDense``, ``miniBatch``, ``momentum``
+     - ``float``
      - The floating-point type that the algorithm uses for intermediate
        computations. Can be float or double.
-   * - method
+   * - ``method``
      - Not applicable
-     - defaultDense
+     - ``defaultDense``
      - Available computation methods:
 
-       - defaultDense
-       - miniBatch
-       - momentum
-   * - batchIndices
-     - defaultDense, miniBatch, or momentum
-     - NULL
+       For CPU:
+
+       - ``defaultDense``
+       - ``miniBatch``
+       - ``momentum``
+
+       For GPU:
+
+       - ``miniBatch``
+
+   * - ``batchIndices``
+     - ``defaultDense``, ``miniBatch``, ``momentum``
+     - ``NULL``
```

Comment:
What NULL means here?

Reply:
This topic was converted from [classical DAAL doc](https://software.intel.com/content/www/us/en/develop/documentation/daal-programming-guide/top/algorithms/analysis/optimization-solvers/iterative-solver/stochastic-gradient-descent-algorithm/computation-9.html).  

Should it be "not applicable" instead of null?

Reply:
fixed